CHANGELOG.md

```diff
@@ -2,9 +2,41 @@
 
 ## Next (unreleased)
 
+## v1.2.0 (May 14, 2019)
+
+### Enhancements
+
+* **Trace detail:** Limit the thickness of spans in the minimap ([@rubenvp8510](https://github.com/rubenvp8510) in [#372](https://github.com/jaegertracing/jaeger-ui/pull/372))
+
+* **Trace detail:** Improve display of long operation names in Operations list ([@kinghuang](https://github.com/kinghuang) in [#351](https://github.com/jaegertracing/jaeger-ui/pull/351))
+
+* **UI Find:** Scroll to first match on load or on press of new locate icon ([@everett980](https://github.com/everett980) in [#367](https://github.com/jaegertracing/jaeger-ui/pull/367))
```

Comment:
Nit: Please use lowercase for follow-on words: `UI find` akin to `Trace detail`

Reply:
SGTM